safenode/src/domain/storage/registers/mod.rs

```diff
@@ -367,6 +457,34 @@ impl RegisterStorage {
             op_log: stored_reg.op_log,
         })
     }
+
+    #[cfg(test)]
```

Comment:
Can we make this function available everywhere by removing the `cfg` flag? This might be useful during the data replication flow.

Reply:
Absolutely, and it has been used like that before. So yep!